api_server/config.yaml.example

```diff
@@ -0,0 +1,9 @@
+## The endpoint on which to serve the Fuel Indexer GraphQL API
+listen_address: 0.0.0.0:29987
+
+## Postgres connection URL. Note that this URI follows the format specified in
+## section 34.1.1.2 of https://www.postgresql.org/docs/current/libpq-connect.html
+database_url: postgres://postgres@127.0.0.1:5432/indexer
+
+## Local path to the local SQLite database
+sqlite_db_path: ../main.db
```

Comment:
Awh, it's a pity the trailing `.example` disables yaml highlighting. Perhaps we should just name this `config.yaml` and then mention its purpose with a comment at the top of the file along the lines of:

```yaml
# The following is an example Fuel Indexer GraphQL configuration.
```